src/Compilers/CSharp/Portable/Errors/ErrorCode.cs

```diff
@@ -771,7 +771,7 @@ internal enum ErrorCode
         ERR_ParamsCantBeRefOut = 1611,
         ERR_ReturnNotLValue = 1612,
         ERR_MissingCoClass = 1613,
-        ERR_AmbigousAttribute = 1614,
+        ERR_AmbiguousAttribute = 1614,
```

Comment:
Hmm, that's not good to have in an API.  Luckily it's a private one.  Have we thought about a spell checker for public APIs?